nkms/client.py

```diff
@@ -80,7 +81,7 @@ def encrypt_key(self, key, pubkey=None, path=None, algorithm=None):
         if not pubkey:
             pubkey = self._pub_key
 
-        if len(path) > 1 and not type(path) is str:
+        if type(path) is tuple and len(path) > 1:
```

Comment:
Instead of checking length first, I check if it's a tuple and then check the length.
This avoided an issue where a `NoneType` `path` was being passed.

However, this can all change with @michwill's propose dchanges where `path` will simply be a string.

Reply:
I think you're right swapping them, yes